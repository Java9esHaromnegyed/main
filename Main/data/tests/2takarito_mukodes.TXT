loadArena testArena2.txt
initArena
addCleanerRobot 0 0 0
tick 7