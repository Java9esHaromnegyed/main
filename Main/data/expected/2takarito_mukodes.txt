arenaLoaded src: testArena2.txt
obstacleAdded type: OIL  pos: [2; 2]
arenaInited
cleanerRobotAdded id: 0 pos: [0; 0]
tick
cleanerRobotMoved id: 0 pos: [0; 1]
oilAged atPos: [2; 2] decay: 1
tick
cleanerRobotMoved id: 0 pos: [0; 2]
oilAged atPos: [2; 2] decay: 2
tick
cleanerRobotMoved id: 0 pos: [1; 2]
oilAged atPos: [2; 2] decay: 3
tick
cleanerRobotMoved id: 0 pos: [2; 2]
obstacleRemoved pos: [2; 2]
tick
cleanerRobotClean id: 0 pos: [2; 2]
tick
cleanerRobotClean id: 0 pos: [2; 2]
obstacleRemoved pos: [2; 2]
